### ISSUE TITLE
Code for batch normalization

### ISSUE THREAD

**Issue by Sahel13** [Orkis-Research/Pytorch-Quaternion-Neural-Networks#12]
Hi,

I was working a research project relating to quaternions, and I would like to use the code in this repository. However, I need to implement batch-normalization for the study, so I wanted to inquire if you have functions implemented for that. In your paper [1] referenced below, you have mentioned that you tried batch-normalization, so that is why I'm inquiring.

[1] Parcollet, T., Morchid, M., & Linarès, G. (2020). A survey of quaternion neural networks. Artificial Intelligence Review, 53(4), 2957–2982. https://doi.org/10.1007/s10462-019-09752-1
